### PR TITLE
global: admin UI first draft

### PR DESCRIPTION
**Pull request by mvesper**
- Uses invenio-search-js to add the search functionality.

### CODE REVIEW

File: invenio_circulation/bundles.py [hunk 1]
```diff
@@ -34,11 +34,13 @@
 )
 
 js = NpmBundle(
+    'node_modules/requirejs/require.js',
     'node_modules/angular/angular.js',
-    'js/circulation/app.js',
+    'js/circulation/circulation.module.js',
     filters='jsmin',
```
Comment (jirikuncar) [inline invenio_circulation/bundles.py]: maybe you need `requirejs` filter instead of including `require.js`

File: invenio_circulation/bundles.py [hunk 2]
```diff
@@ -34,11 +34,13 @@
 )
 
 js = NpmBundle(
+    'node_modules/requirejs/require.js',
     'node_modules/angular/angular.js',
-    'js/circulation/app.js',
+    'js/circulation/circulation.module.js',
```
Comment (jirikuncar) [inline invenio_circulation/bundles.py]: why did you rename it from `app.js`?

File: invenio_circulation/templates/invenio_circulation/index.html [hunk 3]
```diff
@@ -34,6 +34,22 @@
...
 
 {%- block page_body %}
-Hello, World!
+
+<div id="invenio-search">
+  <invenio-search search-endpoint="items/">
+
+    <invenio-search-bar
+     template="{{ url_for('static', filename='templates/invenio_circulation/searchbar.html') }}"
+     placeholder="{{ _('My Uploads') }}">
```
Comment (jirikuncar) [inline invenio_circulation/templates/invenio_circulation/index.html]: My Uploads?

File: invenio_circulation/templates/invenio_circulation/index.html [hunk 4]
```diff
@@ -34,6 +34,22 @@
 {%- endblock javascript %}
 
 {%- block page_body %}
-Hello, World!
+
```
Comment (jirikuncar) [inline invenio_circulation/templates/invenio_circulation/index.html]: can you remove empty lines that are not needed?

File: invenio_circulation/static/js/circulation/app.js [hunk 5]
```diff
@@ -21,9 +21,17 @@
...
 
-(function (angular) {
-  // Bootstrap it!
-  angular.element(document).ready(function() {
-      alert('Hello, World!');
-  });
-})(angular);
+
+require([
+    'node_modules/angular/angular.js',
```
Comment (jirikuncar) [inline invenio_circulation/static/js/circulation/app.js:26]: Do we need `.js`?

**Review by jirikuncar — CHANGES_REQUESTED** [review]

File: invenio_circulation/static/js/circulation/app.js (hunk 5 above)
Comment (mvesper) [reply invenio_circulation/static/js/circulation/app.js:26]: Seems to be necessary :/

File: invenio_circulation/templates/invenio_circulation/index.html (hunk 3 above)
Comment (mvesper) [reply invenio_circulation/templates/invenio_circulation/index.html]: Fixed

File: invenio_circulation/templates/invenio_circulation/index.html (hunk 4 above)
Comment (mvesper) [reply invenio_circulation/templates/invenio_circulation/index.html]: Fixed

**Review by jirikuncar — APPROVED** [review]

File: invenio_circulation/views/ui.py [hunk 6]
```diff
@@ -33,4 +33,5 @@
 @blueprint.route('/', methods=['GET'])
 def index():
     """Circulation index page."""
-    return render_template('invenio_circulation/index.html')
+    return render_template('invenio_circulation/index.html',
+                           config=current_app.config)
```
Comment (jirikuncar) [inline invenio_circulation/views/ui.py]: this should be available by default

**Review by jirikuncar — CHANGES_REQUESTED** [review]

File: invenio_circulation/views/ui.py (hunk 6 above)
Comment (mvesper) [reply invenio_circulation/views/ui.py]: Indeed :)

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**